Tests/UnitTests/Purchasing/Purchases/PurchasesGetOfferingsTests.swift

```diff
@@ -26,6 +26,14 @@ class PurchasesGetOfferingsTests: BasePurchasesTests {
         expect(self.mockOfferingsManager.invokedUpdateOfferingsCacheCount).toEventually(equal(1))
     }
 
+    func testFirstInitializationGetsOfferingsIfAppActiveInCustomEntitlementComputation() {
```

Comment:
⚠️  It was important to add this test because a naive re-ordering of `updateAllCachesIfNeeded` would have meant that this would have now been skipped.